File: backend/src/main/java/com/demo/student_management/dto/lop/LopDetailResponse.java
package com.demo.student_management.dto.lop;

import com.demo.student_management.dto.hocsinh.HocSinhResponse;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class LopDetailResponse {
    private String idLop;
    private String tenLop;
    private Integer siSo;
    private List<HocSinhResponse> hocSinhs;
}

File: frontend/StudentManagement.Shared/Dtos/Lop/LopDetailResponse.cs
using System.Collections.Generic;
using StudentManagement.Shared.Dtos.HocSinh;

namespace StudentManagement.Shared.Dtos.Lop;

public class LopDetailResponse
{
    public string IdLop { get; set; } = string.Empty;
    public string TenLop { get; set; } = string.Empty;
    public int SiSo { get; set; }
    public List<HocSinhResponse> HocSinhs { get; set; } = new();
}
